packages/ckeditor5-image/src/imageresize.js

```diff
@@ -10,6 +10,7 @@
 import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
 import ImageResizeUI from './imageresize/imageresizeui';
```

Comment:
Let's have it as `ImageResizeButtons`. (ref: https://github.com/ckeditor/ckeditor5/issues/7579#issuecomment-655487065)